geowebcache/wms/src/main/java/org/geowebcache/config/GetCapabilitiesConfiguration.java

```diff
@@ -137,11 +137,13 @@ public String getIdentifier() {
     private synchronized List<TileLayer> getTileLayers(boolean reload) throws GeoWebCacheException {
         List<TileLayer> layers = null;
 
-        WebMapServer wms = getWMS();
-        if (wms == null) {
-            throw new ConfigurationException("Unable to connect to " + this.url);
-        }
-
+        WebMapServer wms = null;
+        
+        try {
+            wms = getWMS();
+        } catch (ServiceException | IOException e) {
+            throw new ConfigurationException("Unable to connect to " + this.url + " :"+e.getMessage(), e);
```

Comment:
Could you change the message here ("Unable to connect..."?  It's confusing as the actual error is likely a parse error instead of a connection issue.